specs/CustomDownload.md

```diff
@@ -0,0 +1,771 @@
+  [propget] HRESULT ResultFilePath([out, retval] LPWSTR* resultFilePath);
+
+  /// Sets the `ResultFilePath` property.
+  [propput] HRESULT ResultFilePath([in] LPCWSTR resultFilePath);
+
+  /// Returns an `ICoreWebView2Deferral` object.  Use this operation to
+  /// complete the event at a later time.
+  HRESULT GetDeferral([out, retval] ICoreWebView2Deferral** deferral);
+}
+
+/// Implements the interface to receive `DownloadProgressSizeInBytesChanged`
+/// event.  Use the `ICoreWebView2Download.DownloadProgressSizeInBytes` property
+/// to get the received bytes count.
+
+[uuid(D98F73CE-D94D-4576-A1E0-EB8989A9DBEF), object, pointer_default(unique)]
+interface ICoreWebView2DownloadProgressSizeInBytesChangedEventHandler : IUnknown
+{
+  /// Provides the event args for the corresponding event. No event args exist
+  /// and the `args` parameter is set to `null`.
+  HRESULT Invoke(
+      [in] ICoreWebView2Download* sender, [in] IUnknown* args);
+}
+
+/// Implements the interface to receive `DownloadEstimatedEndTimeChanged` event. Use the
+/// `ICoreWebView2Download.EstimatedEndTime` property to get the new estimated
+/// end time.
+
+[uuid(D46CD703-958B-4100-9F84-4E6C2FC32D57), object, pointer_default(unique)]
+interface ICoreWebView2DownloadEstimatedEndTimeChangedEventHandler : IUnknown
+{
+  /// Provides the event args for the corresponding event. No event args exist
+  /// and the `args` parameter is set to `null`.
+  HRESULT Invoke(
+      [in] ICoreWebView2Download* sender, [in] IUnknown* args);
+}
+
+/// Implements the interface to receive `DownloadStateChanged` event. Use the
+/// `ICoreWebView2Download.State` property to get the current state, which can
+/// be in progress, interrupted, or complete. Find the `InterruptReason` property
+/// on `ICoreWebView2DownloadStateChangedEventArgs`.
+[uuid(74EEBAA9-704B-4A9A-8331-AEBD541CB62C), object, pointer_default(unique)]
+interface ICoreWebView2DownloadStateChangedEventHandler : IUnknown
+{
+  /// Provides the event args for the corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2Download* sender,
+      [in] ICoreWebView2DownloadStateChangedEventArgs* args);
+}
+
+/// Event args for the `DownloadStateChanged` event.
+[uuid(28E3F13A-4DB1-4738-A0A7-FC4B6F6765AD), object, pointer_default(unique)]
+interface ICoreWebView2DownloadStateChangedEventArgs : IUnknown
+{
+  /// The reason why connection with file host was broken.
+  [propget] HRESULT InterruptReason(
+      [out, retval] COREWEBVIEW2_DOWNLOAD_INTERRUPT_REASON* interruptReason);
+}
+
+/// Represents a download process. Gives access to the download's metadata
+/// and supports a user canceling, pausing, or resuming the download.
+[uuid(0C4A07B1-B610-459F-A574-EC253D5EC40D), object, pointer_default(unique)]
+interface ICoreWebView2Download : IUnknown
+{
+  /// Add an event handler for the `DownloadProgressSizeInBytesChanged` event.
+  HRESULT add_DownloadProgressSizeInBytesChanged(
```

Comment:
Don't need to prefix members with class name. Members are already scoped to the class.

Reply:
Remove Download prefixes from all the methods, events, properties in this class.